apps/condo/domains/organization/integrations/sbbol/sync/syncSbbolSubscriptionPaymentRequestsState.js

```diff
@@ -16,16 +16,16 @@ const syncSbbolSubscriptionPaymentRequestStateFor = async (payment, fintechApi)
     const paymentLogString = `ServiceSubscriptionPayment(id=${payment.id}, externalId=${payment.externalId})`
     const { data, error } = await fintechApi.getPaymentRequestState(payment.id)
     if (error) {
-        debugMessage(`Error fetching status for ${paymentLogString}: `, error)
+        logger.error({ message: `Error fetching status for ${paymentLogString}: `, error })
     } else {
         const { keystone: context } = await getSchemaCtx('ServiceSubscriptionPayment')
         if (!data.bankStatus) {
-            console.error('Error: Status response does not contains "bankStatus" field. Something was changed in SBBOL Fintech API.')
+            logger.error({ message: 'Error: Status response does not contains "bankStatus" field. Something was changed in SBBOL Fintech API.' })
         } else {
             if (get(payment.statusMeta, 'bankStatus') !== data.bankStatus) {
                 const status = SBBOL_PAYMENT_STATUS_MAP[data.bankStatus]
                 if (!status) {
-                    console.error(`Error: status "${data.bankStatus}" cannot be mapped to status values of ServiceSubscriptionPayment, because it does not belongs to the list of known statuses. Consider to add it.`)
+                    logger.error({ message: `Error: status "${data.bankStatus}" cannot be mapped to status values of ServiceSubscriptionPayment, because it does not belongs to the list of known statuses. Consider to add it.` })
```

Comment:
it's better to extract status as another context value

Reply:
Changed